server/service/team_policies.go

```diff
@@ -98,7 +98,12 @@ func (svc Service) NewTeamPolicy(ctx context.Context, teamID uint, p fleet.Polic
 /////////////////////////////////////////////////////////////////////////////////
 
 type listTeamPoliciesRequest struct {
-	TeamID uint `url:"team_id"`
+	TeamID                  uint                 `url:"team_id"`
+	Opts                    fleet.ListOptions    `url:"list_options"`
```

Comment:
Does this new argument need to be marked as `optional`?

Reply:
doesn't look like it in my testing, i'm not getting validation errors when sending 0 params